cpp/StatementPutGet.cpp

```diff
@@ -74,6 +74,10 @@ bool StatementPutGet::parsePutGetCommand(std::string *sql,
   {
     putGetParseResponse->stageInfo.stageType = StageType::S3;
     putGetParseResponse->stageInfo.region = response->stage_info->region;
+    // FIPS Support
+    if (response->stage_info->endPoint != NULL) {
```

Comment:
Should you also check that endPoint isn't empty?

Reply:
I can put a check there but it would be redundant because the empty check is present in SnowflakeS3Client.cpp.